Tests/ChoreService_test.py

```diff
@@ -400,6 +400,22 @@ def test_exists(self):
         self.assertTrue(self.tm1.chores.exists(self.chore_name2))
         self.assertTrue(self.tm1.chores.exists(self.chore_name3))
         self.assertFalse(self.tm1.chores.exists(uuid.uuid4()))
+        
+    def test_search_for_process_name_no_match(self):
+        chore_names = self.tm1.chores.search_for_process_name(process_name="NotAProcessName")
+        self.assertEqual([], chore_names)
+
+    def test_search_for_process_name_happy_case(self):
+        chore_names = self.tm1.chores.search_for_process_name(process_name=process_name1)
+        self.assertEqual(self.chore_name1, chore_names[0].name)
```

Comment:
Maybe also assert that chore_names has a length of 1

Reply:
Yes. You are right. process_name1 appears in the tasks for both chores.

Reply:
I think this assertion would be fairly complete.
```  
        chore_names = self.tm1.chores.search_for_process_name(process_name=process_name1)
        self.assertEqual(2, len(chore_names))
        self.assertEqual(self.chore_name1, chore_names[0].name)
        self.assertEqual(self.chore_name2, chore_names[1].name)
```

Reply:
Thanks! i've updated the assertions with this commit 0c1c0d84bdc3bfd8532c6ba9898e41df5740ae9b